scripts/install-tests.sh

```diff
@@ -13,7 +13,11 @@ fi
 if [ "${TESTING_PROFILE}" = "regression-integration-tests" ]; then
     pip3 install -r dockstore-webservice/src/main/resources/requirements/1.13.0/requirements3.txt
 elif [ "${TESTING_PROFILE}" == "language-parsing-tests" ]; then
-    pip3 install -r dockstore-webservice/src/main/resources/requirements/1.14.0/requirements3.txt
+    #pip3 install -r dockstore-webservice/src/main/resources/requirements/1.14.0/requirements3.txt
+    sudo apt-get update
+    # https://stackoverflow.com/questions/44331836/apt-get-install-tzdata-noninteractive needed by cwltool
+    DEBIAN_FRONTEND=noninteractive sudo apt-get install -y tzdata
+    sudo apt-get -qq --yes --force-yes install cwltool=3.1.20220224085855-1
```

Comment:
Probably would be good to make all of these `apt-get` invocations as similar to each other as possible, so if there are functional differences, they are immediately apparent.